package.json

```diff
@@ -85,7 +85,6 @@
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-url": "^2.2.2",
     "semantic-release": "^15.13.12",
-    "styled-components": "^4.1.3",
     "typescript": "^3.2.2"
```

Comment:
I think we still need `styled-components` declared in `devDependencies` so that when running the project locally to develop new components it can find  `styled-components` in `node_modules` 👍